Source/Core/Core/HW/DSPHLE/UCodes/ASnd.h

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <array>
+#include <utility>
+
+#include "Common/CommonTypes.h"
+#include "Core/HW/DSPHLE/UCodes/UCodes.h"
+
+namespace DSP::HLE
+{
+class DSPHLE;
+
+struct ASndUCode : public UCodeInterface
+{
+  ASndUCode(DSPHLE* dsphle, u32 crc);
+  ~ASndUCode() override;
```

Comment:
Why the override?  Why not just use the parent?

Reply:
Because I copy-pasted from #10761; otherwise, it would have looked similar to this:

```C++
AXUCode::~AXUCode()
{
  m_mail_handler.Clear();
}
```

which is actually not useful (since the calling code clears m_mail_handler).  I'll remove it here, and amend #10761 to remove the now-empty destructors.

Reply:
#10761 has been updated.